recipes/xsum/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "xsum" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 21ff547dbb64e8ddefd376f2889349c7a94fce69e7fc417fc661068b298473b1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python 
+    - pip
+    - pybind11
+  run:
+    - python
+    - pybind11
+    - libcxx
```

Comment:
Does xsum use `libcxx` directly?

Reply:
@scopatz No, I do not think so.

Reply:
@scopatz The first time I uploaded the `xsum`, I did not remove the `noarch` and I added `libcxx` to avoid system error.

Reply:
@scopatz I remove libcxx and try it again

Reply:
Yeah, you shouldn't need this explicitly. It is implied by the compiler call above